shiptendance/models/pirate.js

```diff
@@ -0,0 +1,11 @@
+var mongoose = require('mongoose');
+
+var pirateSchema = new mongoose.Schema({
+  name: {type: String, required: true},
+  catchphrase: {type: String, required: true},
```

Comment:
You need to remove the comma at the end of this line since it's the last field you are defining in the schema.